.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
```

Comment:
I am not sure about the dependabot. Do we want to add it?

I mostly worried about this repo being not owned by anybody really so who is gonna handle the PRs?

Reply:
I will adopt it :-)

Reply:
Sounds good :)